source/Sashimi.Tests/ActionHandlersFixture.cs

```diff
@@ -28,6 +28,7 @@
 namespace Sashimi.Terraform.Tests
 {
     [TestFixture(BundledCliFixture.TerraformVersion)]
+    [TestFixture("0.13.0")]
```

Comment:
Let's add a test for v0.11 if it doens't support the `-json` tag to ensure we have covered the functionality in that GetVersion method.

Reply:
I was thinking the bundled version is `0.11.15` (and given our new strategy of BYOT) isn't going to change in the future, but then we still might want to update bundled version to `1.0.0`, in which case our tests will stop covering the supported version range 🤔

Reply:
I really just wanted to ensure the test covered the json and non-json variants of that behaviour more than a specific range of clients.

Reply:
Oh, in that case the bundled version doesn't support `--json` option and works via the fallback.

Reply:
Ah sorry, I didn't really interpret this case properly:

Reply:
> [TestFixture(BundledCliFixture.TerraformVersion)]

Reply:
All good then :)